javascript/javascript2/week3/objectfunction.js

```diff
@@ -0,0 +1,6 @@
+const obj = {
```

Comment:
Find a better name  like bellow :)
```suggestion
const Stundet= {
```